pkg/scheduler/reconciliation/persistent.go

```diff
@@ -403,5 +407,5 @@ func (r *PersistentReconciliationRepository) UpdateOperationState(schedulingID,
 
 		return nil
 	}
-	return db.Transaction(r.Conn, dbOps, r.Logger)
+	return db.Transaction(r.Conn, dbOps, r.Logger) // TODO: Maybe return tx
```

Comment:
is comment still valid: why return tx?

Reply:
not valid, left over from the very beginning